### ISSUE TITLE
[Question]: Evaluating as a multi-label problem: False

### ISSUE THREAD

**Issue by sorth1992** [flairNLP/flair#3283]
### Question

I'm training a SequenceTagger model for NER. My Corpus contains 13 different Labels. Training works and everything goes fine, but the evaluation during epochs seem off. 

The Message "Evaluating as a multi-label problem: False" shows up and I think that's the reason why the F1-Scores are so bad. When I cancel after a few epochs and check the report, the F1-Scores for all entities are above 0.80.

How can I change the evaluation method during epochs? I did not have the problem with flair 0.10, but I can't downgrade because we have scans running on our containers and old version are listed as security threats and thus the containers are deleted.

I am using the latest flair version.

Code looks like this:

`def train_model(data_path, embeddings_path, model_path):

    # GPU-Speicher leeren um Memory-Errors zu vermeiden
    torch.cuda.empty_cache()

    # GPU festlegen
    flair.device = torch.device('cuda:0')

    # Festlegen des Formats der Datensätze, also Text in der ertsen Spalte, Entitäten in der zweiten
    columns = {0: 'text', 1: 'ner'}

    # Setzen des Pfades der Trainings- und Testdateien
    data_folder = data_path

    # Erstellen eines Korpus der das oben angelegte Format der Datensätze und deren Pfade erhält
    corpus: Corpus = ColumnCorpus(data_folder, columns,
                                  train_file='train.txt',
                                  test_file='test.txt',
                                  dev_file='dev.txt')
    label_type = 'ner'

    # Erstellen des Label-Dictionaries. 
    label_dict = corpus.make_label_dictionary(label_type=label_type)
    print(label_dict)

    # Laden der Embeddings
    embeddings = FlairEmbeddings(embeddings_path)

    # Struktur des Modells erstellen
    tagger = SequenceTagger(hidden_size=256,
                            embeddings=embeddings,
                            tag_dictionary=label_dict,
                            tag_type=label_type,
                            use_crf=True)

    # ModelTrainer-Klasse initiieren
    trainer = ModelTrainer(tagger, corpus)

    # Vor Training nach Checkpoint suchen
    folder_content = []
    for cp in os.listdir(model_path):
        folder_content.append(cp)

    if 'checkpoint.pt' in folder_content:
        trained_model = SequenceTagger.load(model_path + '/checkpoint.pt')
        trainer.resume(trained_model,
                       base_path=model_path + '-resume',
                       learning_rate=0.1,
                       mini_batch_size=4,
                       max_epochs=150
                       )
    else:
        # Start des Trainings
        trainer.train(model_path,
                      learning_rate=0.1,
                      mini_batch_size=4,
                      max_epochs=150,
                      checkpoint=True)`

**Comment by sorth1992** [comment]
It seems to be a message that only appeared because I used embeddings that were trained with flair 0.10. I retrained the embedding model and now the message doesn't show up during training and the f1-scores seem plausible
